2025-09-05 15:32:24,207 - large_scale_cognitive_test_client_9 - INFO - Starting client 9 with config: {'client_id': 9, 'server_address': '0.0.0.0:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 465, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:32:29,116 - large_scale_cognitive_test_client_9 - INFO - Client 9 initialized with 6101 training samples
2025-09-05 15:32:29,116 - large_scale_cognitive_test_client_9 - INFO - Client 9 connecting to 0.0.0.0:8080
2025-09-05 15:32:29,116 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:32:29,133 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:32:29,135 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:32:29,139 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:32:29,342 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:32:29,343 - large_scale_cognitive_test_client_9 - ERROR - Client 9 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)", grpc_status:14}"
>
2025-09-05 15:38:41,502 - large_scale_cognitive_test_client_9 - INFO - Starting client 9 with config: {'client_id': 9, 'server_address': '0.0.0.0:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 465, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:38:46,554 - large_scale_cognitive_test_client_9 - INFO - Client 9 initialized with 6101 training samples
2025-09-05 15:38:46,555 - large_scale_cognitive_test_client_9 - INFO - Client 9 connecting to 0.0.0.0:8080
2025-09-05 15:38:46,555 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:38:46,573 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:38:46,574 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:38:46,575 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:38:46,575 - large_scale_cognitive_test_client_9 - ERROR - Client 9 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_status:14, grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"}"
>
2025-09-05 15:41:26,998 - large_scale_cognitive_test_client_9 - INFO - Starting client 9 with config: {'client_id': 9, 'server_address': '20.169.237.127:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 465, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:41:31,681 - large_scale_cognitive_test_client_9 - INFO - Client 9 initialized with 6101 training samples
2025-09-05 15:41:31,682 - large_scale_cognitive_test_client_9 - INFO - Client 9 connecting to 20.169.237.127:8080
2025-09-05 15:41:31,682 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:41:31,700 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:41:31,701 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:41:31,702 - flwr - DEBUG - ChannelConnectivity.CONNECTING
2025-09-05 15:41:31,947 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:41:32,155 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:41:32,155 - large_scale_cognitive_test_client_9 - ERROR - Client 9 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:20.169.237.127:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:20.169.237.127:8080: Failed to connect to remote host: connect: Connection refused (61)", grpc_status:14}"
>
